### PR TITLE
[ViennaRNA] Build script for historic version ViennaRNA-1.8.5

### PR DESCRIPTION
**Pull request by marcom**
This is a build script for ViennaRNA-1.8.5 (from 2011), the last version of ViennaRNA-1.x.y.

I need it for some old projects of mine to make them easily reproducible.

I have placed the script in an extra directory so as not to mess up the build script for the current ViennaRNA-2.x.y.

### CODE REVIEW

File: V/ViennaRNA/ViennaRNA-1.8.5/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    ExecutableProduct("RNApaln", :RNApaln),
+    ExecutableProduct("RNApdist", :RNApdist),
+    ExecutableProduct("RNAplfold", :RNAplfold),
+    ExecutableProduct("RNAplot", :RNAplot),
+    ExecutableProduct("RNAsubopt", :RNAsubopt),
+    ExecutableProduct("RNAup", :RNAup),
+    LibraryProduct("libRNA", :libRNA),
+]
+
+dependencies = [
```
Comment (giordano) [inline V/ViennaRNA/ViennaRNA-1.8.5/build_tarballs.jl:77]:
```suggestion
dependencies = Dependency[
```

**Comment by marcom** [comment]
https://buildkite.com/julialang/yggdrasil/builds/3762#0188ff37-685c-4f07-a671-a37bf946e67f/2247-2248

Is the build_tarballs.jl being under an extra subdirectory (ViennaRNA-1.8.5) somehow messing up the tarball name under products?

Tarball is `ViennaRNA.v1.8.5.armv6l-linux-gnueabihf.tar.gz`

But then this fails:
```
$ buildkite-agent artifact upload **/products/ViennaRNA-1.8.5*.tar.gz
2023-06-27 23:38:23 INFO   No files matched paths: **/products/ViennaRNA-1.8.5*.tar.gz
```

**Comment by marcom** [comment]
I somehow messed up the subdir renaming, now ViennaRNA-2.6.2 is being built...
I'll open a new PR.

**Comment by marcom** [comment]
Continued in #6951

**Pull request closed without merging**